library/Notifications/Widget/Timeline/Entry.php

```diff
@@ -50,23 +51,23 @@ public function getColor(int $transparency): string
     /**
      * Set content of a tooltip that is shown when the entry is hovered
      *
-     * @param ValidHtml $content
+     * @param HtmlDocument $content
      *
      * @return static
      */
-    public function setFlyoutContent(ValidHtml $content): static
+    public function setFlyoutContent(HtmlDocument $content): static
     {
-        $this->flyoutContent = $content;
+        $this->flyoutContent = clone $content;
```

Comment:
With the new widget, consider to avoid cloning it here again. Think about a factory method, since the current member is only known later on anyway. Assembling an entry is already responsible to inject the flyout content, so I'd suggest to add a method like `withCurrentMember(Member $member): self` and inject its result instead. (Still inside `assembleContainer`)